spec/sword/endpoints/academic_commons_endpoint_spec.rb

```diff
@@ -35,10 +35,6 @@
         # for test purposes, turn off actual sending of POST request
         @ac_endpoint.hyacinth_adapter.no_op_post = true
       end
-
-      it '#handle_deposit is called successfully (without crashing)' do
```

Comment:
Based on the other changes in this PR, it's not clear to me why this test was removed.

Reply:
This test is brittle and not very useful. If required, I can create a ticket to add the test to a future release.

Reply:
I added the simplecov gem to this branch for exactly this kind of case!  If you run rspec on your local machine it will now create a (gitignored) "coverage" directory, and that shows which lines of code are covered by specs and which aren't.  I just tried running it and this was the output for this method:

![Screen Shot 2024-11-01 at 5 39 07 PM](https://github.com/user-attachments/assets/9f57858d-554c-4ee9-828b-7bf9bafeb52c)

I checked Jira and it looks like you already have a ticket for improving test coverage in the future (SWORD-100), so maybe worth bundling this test improvement with other spec improvements in that ticket?

Reply:
Added a comment to SWORD-100 to deal with coverage issues logged by simplecov.